src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,61 @@
+package coffee.order;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+    private Queue<Order> orders;
+    private int nextOrderNumber;
+
+    public CoffeeOrderBoard() {
+        this.orders = new LinkedList<>();
+        this.nextOrderNumber = 1;
+    }
+
+    public void add(String name) {
+        Order order = new Order(name, nextOrderNumber);
+        orders.add(order);
+        nextOrderNumber++;
+    }
+
+    public Order deliver() {
+        return orders.poll();
+    }
+
+    public Order deliver(int nextOrderNumber) {
+        Order order = null;
+        for (Order element : orders) {
+            if (element.getNumber() == nextOrderNumber) {
+                order = element;
+                break;
+            }
+        }
+        if (order != null) {
+            orders.remove(order);
+        }
+        return order;
+    }
+
+    public void draw() {
+        System.out.println("Num\t| Name");
```

Comment:
Remember, I've told you not to use print statements? You can use Logger. Another solution would be to make this method return a string with all the info you need and print it in `main` method.